src/libmagickwand.jl

```diff
@@ -25,117 +25,15 @@ else
     error("ImageMagick not properly installed. Please run Pkg.build(\"ImageMagick\") then restart Julia.")
 end
 
-const libmagick = Ref{Ptr{Void}}()
-
-const MagickWandGenesis                = Ref{Ptr{Void}}()
```

Comment:
In light of #93, I'd like it if @musm or @Allardvm would comment here.

Reply:
I'm pretty sure that we don't need those shenanigans anymore, now that the windows binaries are pretty much identical to the unix ones - instead of using this weird windows installer, that messes with the registry and loads everything dynamically from an xml file^^ 
And I've been mainly testing this on windows and appveyor passes, so this might just be the end to windows being crazy with ImageMagick :)